pylintrc

```diff
@@ -0,0 +1,57 @@
+[MASTER]
```

Comment:
Doing a cursory comparison between this and the pylintrc in the AML repo, I didn't see any major differences, although I didn't compare the longer lines perfectly. Is there any big changes pylint's expected formatting that is implied here?

Reply:
Nope, you're right. It's pretty much exactly the same as AzureML's pylintrc - there are just a few pylint errors that I excluded that are a bit Azure SDK-specific like client-accepts-api-version-keyword,networking-import-outside-azure-core-transport, and missing-client-constructor-parameter-credential.